app/features/modelling/line-plot/index.js

```diff
@@ -8,24 +40,25 @@ class LinePlotModel {
     this.frame = frame;
     fetch(`${src}?=`)
       .then(response => response.json())
-      .then((data) => {
-        let datasets;
-        const { chartOptions, data: FEMdata, ...rest } = data;
-        if (FEMdata) {
-          datasets = FEMdata.map( (data, index) => {
-            const { seriesData, seriesOptions } = data
-            return {
-              data: seriesData,
-              label: seriesOptions.label || `Series ${index + 1}`,
-              pointStyle: seriesOptions.glyph,
-              backgroundColor: seriesOptions.color,
-              borderColor: '#000000'
+      .then(data => {
+        if(data.x && data.y) {
+          return {
+            data: {
+              x: data.x,
+              y: data.y
             }
-          });
-        }
-        if (data.datasets) {
-          datasets = data.datasets
+          },
+          {
+            chartOptions: {
+              xAxisLabel: 'Days',
+              yAxisLabel: 'Brightness'
+            }
+          };
```

Comment:
This converts PH-TESS subjects from `{ x, y }` to `{ data: { x, y }, chartOptions }` (when the location MIME type is `application/json`.)